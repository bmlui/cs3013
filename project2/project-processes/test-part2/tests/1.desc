Student code should not use system(), but it should include fork() and exec()
