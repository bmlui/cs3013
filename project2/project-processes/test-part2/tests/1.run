grep "system(" ../wshell.c; grep "exec.\{1,\}(" ../wshell.c -l; grep "fork(" ../wshell.c -l; objdump -d wshell | grep "exec.\{1,\}@plt" -l
