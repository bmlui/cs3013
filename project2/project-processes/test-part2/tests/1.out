../wshell.c
../wshell.c
(standard input)
